python/cog/server/http.py

```diff
@@ -400,7 +416,7 @@ def _signal_set_event(signum: Any, frame: Any) -> None:
     s.start()
 
     try:
-        shutdown_event.wait()
+        asyncio.run(shutdown_event.wait())
```

Comment:
```suggestion
        shutdown_event.wait()
```

unless this was thoroughly tested